test/dlc_tests/conftest.py

```diff
@@ -27,7 +27,7 @@
     "tensorflow_inference",
     "tensorflow_training",
     "training",
-    "inference"
+    "inference",
```

Comment:
Revert this line please.

Reply:
Yes. The reformatting is implemented by command